.github/workflows/documentation.yml

```
@@ -45,6 +45,30 @@ jobs:
           python-version: ${{ env.PYTHON_VERSION }}
           cache: pip
 
+      - name: Set version variables
+        run: |
+          cd grass
+          eval $(./utils/update_version.py status --bash)
+          echo "MAJOR=$MAJOR" >> $GITHUB_ENV
+          echo "MINOR=$MINOR" >> $GITHUB_ENV
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "YEAR=$YEAR" >> $GITHUB_ENV
+
+      - name: Checkout addons
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: OSGeo/grass-addons
+          ref: grass${{ env.MAJOR }}
+          path: grass-addons
+
+      - name: Install Python dependencies
+        # We install both core and addon dependencies, but we don't install any
+        # dependencies from the extra list in addons because we don't actually
+        # run them (and they should be lazy-imported).
+        run: |
+          pip install -r grass/.github/workflows/python_requirements.txt
+          pip install -r grass-addons/.github/workflows/python_requirements.txt
```

That file doesn't exist. It's either

```suggestion
          pip install -r grass-addons/.github/workflows/requirements.txt
```

Or, once a env var is set for gdal (`GDAL_VERSION`):
```suggestion
          pip install -r grass-addons/.github/workflows/extra_requirements.txt
```
(That also includes the base requirements.txt)

---

I'll do just the basic ones. See comment. It is all that should be needed for doc.